File: composicion-agregacion/Ejercicio12_java/Hospital.py
class Hospital:
    def __init__(self, nombre):
        self.__nombre = nombre
        self.__doctores = []
        self.__cantDoctores = len(self.__doctores)
    
    def agregarDoctor(self, dr):
        self.__doctores.append(dr)
    
    def mostrarDoctores(self):
        print(f"*** {self.__nombre}\n<<doctores>>")
        for d in self.__doctores:
            print(d)

File: composicion-agregacion/Ejercicio12_java/java/Hospital.java
import java.util.ArrayList;
import java.util.List;

class Hospital {
    private String nombre;
    private List<Doctor> doctores;

    public Hospital(String nombre) {
        this.nombre = nombre;
        this.doctores = new ArrayList<>();
    }

    public void agregarDoctor(Doctor dr) {
        doctores.add(dr);
    }

    public void mostrarDoctores() {
        System.out.println("*** " + nombre + "\n<<doctores>>");
        for (Doctor d : doctores) {
            System.out.println(d);
        }
    }
}
